grakn/rpc/database_manager.py

```diff
@@ -37,11 +39,11 @@ def create(self, name: str) -> None:
         pass
 
     @abstractmethod
-    def delete(self, name: str) -> None:
```

Comment:
`delete` no longer exists in `DatabaseManager`. It is now a method on `Database`.